bot/util/utilities.py

```diff
@@ -1,29 +1,158 @@
-"""Contains a cog for all utility funcionality"""
+"""Contains a Cog for all utility funcionality."""
+
+from datetime import datetime
 from discord.ext import commands
+import discord
 
 
 class UtilitiesCog(commands.Cog):
-    """Cog for utility functions"""
+    """Cog for Utility Functions."""
 
     def __init__(self, bot):
-        """Initializes the Cog"""
+        """Initializes the Cog."""
         self.bot = bot
 
     @commands.command(name='ping')
     async def ping(self, ctx):
-        """
-        Command Handler for the Ping Command
-        Responds with a message containing 'Pong!', as well as the latency to the discord server in ms.
+        """Command Handler for the `ping` command.
+
+        Args:
+            ctx (Context): The context in which the command was called.
 
-        :param ctx: the context in which the command was called
+        Returns:
+            str: A message containing 'Pong!', as well as the measured latency to the Discord server in milliseconds.
         """
         latency = round(self.bot.latency * 1000, 2)
-        await ctx.send("\U0001F3D3 **Pong!** - {0} ms".format(latency))
+        await ctx.send(":ping_pong: **Pong!** - {0} ms".format(latency))
+
+    @commands.command(name='serverinfo')
+    async def server_info(self, ctx):
+        """Command Handler for the `serverinfo` command.
+
+        Args:
+            ctx (Context): The context in which the command was called.
+
+        Returns:
+            Embed: An embedded message (Embed) containing a variety of stats and interesting information about the
```

Comment:
Be more specific than "a variety of stats". Tell, what stats and insights this message contains